db/migrate/20150806211453_rename_ems_event_table_to_event_stream.rb

```diff
@@ -13,7 +14,13 @@ def up
 
     add_column :event_streams, :type, :string
     say_with_time("Updating Type in EventStreams") do
-      EventStream.update_all(:type => 'EmsEvent')
+      update_count = nil
+      done = 0
+      until update_count == 0
+        update_count = EventStream.where(:type => nil).limit(1000).update_all(:type => 'EmsEvent')
+        done += update_count
+        say "#{done.to_s.reverse.gsub(/...(?=.)/, '\&,').reverse} rows", :subitem if update_count > 0
```

Comment:
@gmcculloug we did in the past, I don't see any now currently.

I think this say is a progress indicator.

How about a say at the beginning of the batches, saying how many there are and how many we're doing in each batch... and each batch print "." as a progress indicator.  Just an idea, I don't know how it would look.